client/src/App.js

```diff
@@ -1,27 +1,37 @@
 import styled from 'styled-components/macro'
 import Cards from './components/Cards/Cards'
 import Form from './components/Form/Form'
-import localStorageInit from './components/LocalStorage/LocalStorageInit'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import { nanoid } from 'nanoid'
 import LoginForm from './components/LoginForm/LoginForm'
+import getCards from './services/GetCards'
+import pushCard from './services/PushCard'
 
 function App() {
-  localStorageInit()
-
-  const [data, setData] = useState(JSON.parse(localStorage.getItem('data')))
+  const [cards, setCards] = useState([])
   const [user, setUser] = useState('')
 
+  useEffect(() => {
+    getCards()
+      .then(data => setCards(data))
+      .catch(error => console.error(error))
+  }, [])
+
   const addQuestionHandler = question => {
-    const newData = [...data, { text: question, author: user, id: nanoid() }]
-    localStorage.setItem('data', JSON.stringify(newData))
-    setData(newData)
+    const newCard = { text: question, author: user, id: nanoid() }
```

Comment:
Mongo automatically creates an `_id` for our cards, so we don't need to set it here.
```suggestion
    const newCard = { text: question, author: user }
```